src/store/external-services/external-services.effects.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+const uri = 'https://bws.bitpay.com/bws/api';
+
+export const getExternalServicesConfig = async () => {
+  try {
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    const {data} = await axios.get(uri + '/v1/services', config);
+
+    return Promise.resolve(data);
+  } catch (err) {
+    console.log(err);
```

Comment:
maybe this could be improved

Reply:
I removed this console.log. It is not necessary